lib/files.cf

```diff
@@ -428,7 +428,7 @@ bundle edit_line manage_variable_values_ini(tab, sectionName)
       ifvarclass => "edit_$(cindex[$(index)])";
 
       # match a line starting like the key something
-      "$(index)\s*=.*"
+      "\s*$(index)\s*=.*"
       edit_field => col("=","2","$($(tab)[$(sectionName)][$(index)])","set"),
```

Comment:
@nickanderson there is an inconsistency here (or at least I think so). The separator regex is not the same for `manage_variable_values_ini` and `set_variable_values_ini`. Where in the former case, there is no whitespace matching, while the latter does add this. I guess this should be standardized also?

This is what results in the weird output you can see in `set_variable_values_ini.cf.finish`.

Reply:
Is this comment outdated?

The regular expressions look the same to me:

```
manage_variable_values_ini
     "\s*$(index)\s*=.*"
          edit_field => col("=","2","$($(tab)[$(sectionName)][$(index)])","set"),

set_variable_values_ini
     "\s*$(index)\s*=.*"
      edit_field => col("\s*=\s*","2","$($(tab)[$(sectionName)][$(index)])","set"),
```

Reply:
> ```
> manage_variable_values_ini
>      "\s*$(index)\s*=.*"
>           edit_field => col("=","2","$($(tab)[$(sectionName)][$(index)])","set"),
> 
> set_variable_values_ini
>      "\s*$(index)\s*=.*"
>       edit_field => col("\s*=\s*","2","$($(tab)[$(sectionName)][$(index)])","set"),
> ```

The regexes are the same here, but:
>           edit_field => col("=","2","$($(tab)[$(sectionName)][$(index)])","set"),
and
>       edit_field => col("\s*=\s*","2","$($(tab)[$(sectionName)][$(index)])","set"),
differ for the `=`, and `\s*=\s*` separators. 

I'm thinking this is not on purpose. So I guess I should unify it?

Reply:
I see. I am not against unifying them, but I don't have an opinion on which way they ought to be unified. I guess I don't care so long as the test captures any subtle change in behavior.

Reply:
Leaving it as a follow up.